realtime_trains/extract.py

```diff
@@ -5,10 +5,30 @@
 from datetime import datetime, timedelta
 import logging
 
+from dotenv import load_dotenv
 from requests.auth import HTTPBasicAuth
 from requests.exceptions import RequestException
 from requests import get
-from dotenv import load_dotenv
+
+import psycopg2
+import psycopg2.extras
```

Comment:
```suggestion
from psycopg2.extras import DictCursor
```

Reply:
Have you considered RealDictCursor?

Reply:
Nope, think it's better if i used it for API development since it's easier to get json data from RealDictCursor then Dictcursor but not in this scenario